code/datums/net/base/net_atom.dm

```diff
@@ -0,0 +1,12 @@
+/atom
+	var/list/net_nodes = list() //list of nodes
+
+/atom/New(loc)
+	. = ..(loc)
```

Comment:
This does nothing, I think

Reply:
needed since there are multiple /atom/New()'s in the code, apparently they dont get called if you dont supercall

Reply:
No, he's right. This  does nothing.

Reply:
it does, i pinged you in discord